### ISSUE TITLE
All Nuke OCIO nodes should have a context tab

### ISSUE THREAD

**Issue by bxyldy** [AcademySoftwareFoundation/OpenColorIO#443]
...unless there's a reason some nodes can't direct the context for the transform?

For instance, our configs make heavy use of the FileTransform's ability to recurse through the search path to find a file that matches if just the basename of a file is provided; and the context variables dictate the search path taken to find said file. It's very convenient to lay down an OCIOFileTransform  node in Nuke, and populate the file path with the word "shot", and have that node automagically find the appropriate LUT named "shot" for the environment in which Nuke was started; but this isn't as useful if I need to see what's going on with the "shot" LUT in a different context.

Similarly, that OCIOLogConvert node would be super handy, if only the "grading" role didn't point to a similarly dynamic colorspace; i.e., for shows that use multiple camera setups and aren't on board with ACES, grading often takes place in the camera's native log / gamut of choice -- so our "grading" role is contingent on the camera, and is therefore subject to change, depending on the context.

OCIOCDLTransforms -- Same thing, if I'm using multiple .ccc files in the path (cuz i'm either a masochist, or i'm using the cccid to differentiate between versions of grades, or different plate elements for the same shot, or... other stuff). 

In summary: I dig context tabs.

**Comment by dbr** [comment]
I can't find the discussion now, but there was no particular reason the OCIOLogConvert excluded the Context tab - was more just a case of there being no obvious use case for them at the time (e.g generally `compositing_log` was expected to be static)

Since this sounds like a reasonable workflow, adding the tabs to the other nodes seem reasonable..

We have had the same requirement in the past (per-shot log transforms), in which case we just used the OCIOColorSpace node with a "per_shot_log" type colourspace. Would have been nicer if the OCIOLogConvert node would have worked.
